src/dll/Hooks/ExecuteProcess.cpp

```diff
@@ -3,30 +3,62 @@
 #include "App.hpp"
 #include "Hook.hpp"
 #include "Systems/ScriptCompilationSystem.hpp"
+#include <winbase.h>
 
 namespace
 {
 bool isAttached = false;
 
-bool _Global_ExecuteProcess(void* a1, RED4ext::CString& aCommand, FixedWString& aArgs,
-                            RED4ext::CString& aCurrentDirectory, char a5);
+bool _Global_ExecuteProcess(ScriptCompilation* a1, RED4ext::CString& aCommand, FixedWString& aArgs,
```

Comment:
Is this a `Global` or is this a method for `ScriptCompilation` class?

Reply:
`ScriptCompilation::ExecuteProcess` 🏃

Reply:
Looking at it more in the game, it's definitely a method for the class, but I don't think the class is unique to script compilation - I'll dig a bit more & put it in the SDK.